assets/app/scripts/controllers/overview.js

```diff
@@ -180,10 +181,17 @@ angular.module('openshiftConsole')
 
       // Hide our deployer pods since it is obvious the deployment is
       // happening when the new deployment appears.
+
+      // Check for the deployer pod annotation.
       if (annotationFilter(pod, "openshift.io/deployment.name")) {
         return false;
       }
 
+      // Check for the deployer pod label.
+      if ($filter('label')(pod, "openshift.io/deployer-pod-for.name")) {
```

Comment:
The label is relatively new and is used to help get grouped/filtered lists of all deployer pods related to a deployment (of which there could be many: the primary deployment process, pre/post hook pods, custom pods launched by a custom deployment process which conform to the spec and label their pods, etc.)

The annotation which correlates a deployer pod to its deployment isn't going away.